arch/arm/soc/atmel_sam/sam4s/Kconfig.soc

```diff
@@ -112,7 +105,15 @@ config SOC_ATMEL_SAM4S_PLLB_DIVB
 	Board config file can override this settings
 	for a particular board.
 
-	With default of MULB == 6, and DIVB == 1,
-	PLL is running at 7 times of main clock.
+	With default of MULB == 3, and DIVB == 1,
+	PLL is running at 4 times of main clock.
+
+config SOC_ATMEL_SAM4S_EFC0_FMR_FWS
```

Comment:
Okay, so would it be acceptable to remove the config option and just have it fixed to 5, which is 120 MHz, VDDCORE 1.2V, VDDIO 2.7-3.6 V? Those are the values for the SAM4S Xplained.